main.tex

```diff
@@ -392,18 +392,20 @@ \subsection{Measuring Extortion from the History of Play}
 becomes:
 
 \begin{equation}
-    -x^*_2 < x^*_1 \label{eqn:measured_condition_for_chi}
+    \frac{-x^*_2}{x^*_1} > 1
```

Comment:
Was this better before, since the numerator could be zero?

Reply:
I'm not actually sure. As it was before is technically not correct when $x_1^*$ is negative. I believe $\chi$ itself is not defined for $x_1^*=0$. I'll add a note about that. Let me know what you think though.

Reply:
hmm yeah $\alpha=0$ or $\beta=0$ are special/unusual cases.

Reply:
Maybe $\frac{-x^*_2}{x^*_1} = \Chi > 1$ more clearly connects the $x$ values, $\Chi$, $\alpha$, and $\beta$

Reply:
Yeah good idea, that is better.

Reply:
[5256bed](https://github.com/drvinceknight/testing_for_ZD/pull/133/commits/5256bed5ea937360d7a7a34bf5bb86382998ebf4)